tabula/io.py

```diff
@@ -1,20 +1,21 @@
-"""This module is a wrapper of tabula, which enables extract tables from PDF.
+"""
```

Comment:
This line break isn't mandatory since this line doesn't over 88 characters.
https://github.com/chezou/tabula-py/blob/7a768d9fe2d9d50cdb78b3755e115cde4bbcb383/setup.cfg#L49

Could you double check your configuration for flake8?

Reply:
I just did the required changes and everything seems to be fine now. I also checked max-line-length = 88 in tabula-py\setup.cfg, which is correct.